src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/blasting.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:blasting",
+  "text": [
+    {
+      "text": "This modifier might blow you away if you are not careful."
+    }
+  ],
+  "effects": [
+    "Mines blocks weaker to explosives faster",
```

Comment:
I give an example in the tooltip of ores, because not everyone knows ore are weak to explosions.